.nuget/FStar.nuspec

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
I would comment on `nuspec` usage. Is this file really nescessary? Maybe that can be done with new MSBuild-SDK project files? Also I would like to propose slightly different overall solution

- `Microsoft.FStar.Sdk` SDK nuget which will automatically bring nescessary `props` and `targets` files
- `runtime.win-x64.FStar`, `runtime.linux-x64.FStar` and `runtime.osx-x64.FStar` which will contains Windows, Linux and OS X executables respectively. That's needed for `Microsoft.FStar.Sdk` to be able transparently build on all platforms.

Maybe that's require some plumbing with CI which I can help do, but that would be entirely separate discussion.

Overall all that would be left is create template project and new project can be installed by running simply `dotnet new fstarconsole` and developer can start playing with FStar in the familiar environment.

Reply:
I totally wouldn't mind, I'm not sure if the approach you describe was available when I was working on this. At least I wasn't aware of it.

On a second note, I suggest you discuss it first with @nikswamy before doing any real work.